.github/workflows/release-python-client.yaml

```diff
@@ -0,0 +1,55 @@
+name: Publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+
+on: push
```

Comment:
Oh! And I think we'd want more control over when this runs. We could change this to match the criteria for kicking off the [dev-image-update.yml workflow](https://github.com/DataBiosphere/jade-data-repo/blob/develop/.github/workflows/dev-image-update.yaml#L7).  

Or, we could dispatch this workflow as a part of that action! (I think would prefer this option!) This would mean two changes:
1. 
```suggestion
on:
  workflow_dispatch: {}
```
2. Add a step to dev-image-update.yaml (disclaimer: I've never used the input/output format, but it might be helpful in passing the semantic version if you can get it to work!)
```
- name: "Publish python package"
   uses: broadinstitute/workflow-dispatch@v1
   with:
      workflow: Publish Python 🐍 distributions 📦 to PyPI and TestPyPI
      token: ${{ secrets.WORKFLOW_DISPATCH }}
      inputs: '{ "current_semver": ${{ steps.bumperstep.outputs.tag }} }'  
```
The only trick here is that the version would be a bit behind. The semver step below will grab what's currently running on dev. But, that version bump in dev isn't instant, so it won't pick up the latest version. It looks like you might be able to pass the latest version from dispatch step in dev-image-update using output/input? https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_dispatch   
* info on using the [workflow-dispatch step](https://github.com/broadinstitute/workflow-dispatch)